.planning/phases/00-performance-fix/00-06-PLAN.md

```
@@ -0,0 +1,168 @@
+## Trust Boundaries
+
+| Boundary | Description |
+|----------|-------------|
+| tests→SQLite DB | Test data with known timestamps |
+
+## STRIDE Threat Register
+
+| Threat ID | Category | Component | Disposition | Mitigation Plan |
+|-----------|----------|-----------|------------|-----------------|
+| T-00-07 | Information Disclosure | Test database | accept | Local temp database, isolated to test |
+</threat_model>
```

markdownlint-cli2 (0.22.0)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)